docs/installation.rst

```diff
@@ -1,5 +1,145 @@
 ============
 Installation
 ============
+Installation documentation for Noggin.
 
-.. note:: **TODO**: Cover end-user installation here.
+
+============
+Requirements
+============
+Noggin requires the following to operate:
+
+1. Access to a FreeIPA/Red Hat Identity Management installation
+2. Access to an Openshift cluster to run the Noggin container
+
+
+===================================
+Install Red Hat Identity Management
+===================================
+For more information see the following documents regarding installation
+
+1. .. _Docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/preparing-the-system-for-ipa-server-installation_installing-identity-management
+2. .. _Playbook install docs: https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/installing_identity_management/installing-an-identity-management-replica-using-an-ansible-playbook_installing-identity-management
+
+
+Install ansible and ansible-freeipa rpms
+
+::
+    subscription-manager repos --enable ansible-2.8-for-rhel-8-x86_64-rpms
```

Comment:
Please do not use RHEL as the sole reference documentation for how to install and deploy this. For one, stuff like `subscription-manager` and access to various RH product subscriptions are not going to be available by most people. It's fine to have RHEL specific notes, but please write the installation process using CentOS and Fedora instead, since that's what pretty much everyone has access too.

Reply:
Yes I agree, RHEL installation docs should not be the sole reference but it will be the first documented reference. We've made a decision to first document how we plant to install Noggin in Fedora.

Reply:
Then don't use ansible-for-rhel and use ansible from EPEL.

Reply:
I think I've addressed this @Conan-Kudo but I'm not sure, mind taking another look when you get a chance thanks.